src/core/less/variables/animations.less

```diff
@@ -0,0 +1,32 @@
+// Animations
```

Comment:
do we need a vote on whether this is core?

Reply:
If your wondering about it, its probably good to go.
Animation is a nicety that developers can do on their own theme I guess.
Thanks, removed.